dtcli/scripts/dt.py

```diff
@@ -254,7 +280,37 @@ def genca(**kwargs):
     )
 
 
-_deprecate_above, _deprecate_below = deprecated("dt ext generate-developer-pem")
+@typer_extension.command()
+def generate_certificate_authority(
+    ca_cert: str = typer.Option(
+        const.DEFAULT_CA_CERT,
+        help="CA certificate output path"
+    ),
+    ca_key: str = typer.Option(
+        const.DEFAULT_CA_KEY,
+        help="CA key output path"
+    ),
+    ca_subject: List[str] = typer.Option(
+        ["CN=Default Extension CA", "O=Some Company", "OU=Extension CA"],
+        help='Certificate subjects. Accepted format is "key=value"'
+    ),
+    force: bool = FORCE_OPTION,
+    days_valid: int = typer.Option(
+        const.DEFAULT_CERT_VALIDITY,
+        help="Number of days certificate will be valid"
+    ),
+):
+    """
+    Creates CA key and certificate, needed to create developer certificate used for extension signing.
+    """
+    _genca(
+        ca_cert_path=ca_cert,
+        ca_key_path=ca_key,
+        force=force,
+        subject=validate_ca_subject(ca_subject),
```

Comment:
It would be more elegant to do it as a callback, but typer automatically converts it to List[str] I can't find a workaround